Assets/HoloToolkit/Utilities/Editor/SceneLauncherEditor.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.SceneManagement;
+
+namespace HoloToolkit.Unity
+{
+    [CustomEditor(typeof(SceneLauncher))]
+    public class SceneLauncherEditor : Editor
```

Comment:
This just makes it easier to add scenes to the build settings without having to go through the build window.